packages/core/src/node/mdx/remarkPlugins/checkDeadLink.ts

```diff
@@ -66,9 +65,9 @@ export const remarkCheckDeadLinks: Plugin<
       }
 
       if (!url.startsWith('http') && !url.startsWith('https')) {
-        const normalizeUrl = normalizeRoutePath(
+        const { routePath: normalizeUrl } = routeService.normalizeRoutePath(
           // fix: windows path
-          url?.split(path.sep).join('/')?.split('#')[0],
+          url.split(path.sep).join('/')?.split('#')[0],
```

Comment:
FYI, I've removed optional operator here as from TS definitions it didn't seem to be needed